libqtile/scripts/qtile.py

```diff
@@ -29,7 +29,11 @@
 from libqtile import confreader
 from libqtile.backend.x11 import xcore
 
-locale.setlocale(locale.LC_ALL, locale.getdefaultlocale())  # type: ignore
+try:
+    locale.setlocale(locale.LC_ALL, locale.getdefaultlocale())  # type: ignore
+except locale.Error:
+    pass
```

Comment:
Maybe log something?

Reply:
I thought about that, but we don't have the log set up here yet. By the time we do, some of the locale stuff may already be processed.

Reply:
We could figure it out if we need, but I think the only reason we use the locale right now is for currency in the bitcoin/stock ticker widget, so it's pretty low priority :)

Reply:
I have tried this in the live usb - does not seem to help

Reply:
Afterwards the last line seems to be different - confreader
![error-code](https://user-images.githubusercontent.com/36679406/78284120-30b7a880-751f-11ea-8da7-98da4bc65235.png)

Reply:
I confirm this code would fix qtile crashing on ArcoLinux when used together on iso and lightdm.

Reply:
#1581 See trials and successful installation